### PR TITLE
Show detailed public tracking events via the tracking api

### PR DESCRIPTION
**Pull request by taryneast**
This is the api-doc update to match this ticket:

https://trello.com/c/tGtTrQPO

And this code PR:
https://github.com/sendle/sendle/pull/3330/files

Changes:
Add description of new fields from tracking changes
Also add an example of a non-tracking public event to the list of tracking changes

### CODE REVIEW

**Comment by taryneast** [comment]
[Show the detailed public tracking events via the tracking API](https://trello.com/c/tGtTrQPO/47-show-the-detailed-public-tracking-events-via-the-tracking-api)

File: source/includes/_track_parcel.md [hunk 1]
```diff
@@ -14,7 +14,13 @@
     "state":"Delivered",
     "tracking_events":
       [
-        {
+       {
+          "event_type":"Pickup Attempted",
+          "scan_time":"2015-11-23T01:04:00Z",
```
Comment (seangeo) [inline source/includes/_track_parcel.md]: "scan_time" is some unfortunate naming :(

Comment (taryneast) [reply source/includes/_track_parcel.md]: yeah :(

**Comment by taryneast** [comment]
The naming matches what's on the public tracking page (for good for ill).
We could do a renaming in another ticket if you like?

**Comment by seangeo** [comment]
Do we really need another ticket? Can we just change it now?

**Pull request merged**